tests/baselines/reference/outModuleConcatES6.js.map

```diff
@@ -1,2 +1,2 @@
 //// [all.js.map]
-{"version":3,"file":"all.js","sourceRoot":"","sources":["tests/cases/compiler/ref/a.ts","tests/cases/compiler/b.ts"],"names":[],"mappings":"AACA,0BAA0B"}
\ No newline at end of file
+{"version":3,"file":"all.js","sourceRoot":"","sources":["tests/cases/compiler/ref/a.ts","tests/cases/compiler/b.ts"],"names":[],"mappings":"AACA,0BAA0B;AAE1B,MAAM,OAAO,CAAC;CAAI;ACHlB,OAAO,EAAC,CAAC,EAAC,MAAM,SAAS,CAAC;AAC1B,MAAM,OAAO,CAAE,SAAQ,CAAC;CAAI"}
```

Comment:
This is another case where I'd rather we not accept this file as it uses `outFile`.